packages/web/src/forms/CheckboxWithLabel.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react'
+import { createElement, StyleSheet, Text, View } from 'react-native'
+import { colors, fonts, standardStyles, textStyles } from 'src/styles'
+import { Label } from './FormComponents'
+
+interface CheckboxProps {
+  checked: boolean
+  name: string
+  onPress: (x: any) => void
+}
+
+export function CheckboxWithLabel({
+  checked,
+  onPress,
+  name,
+  label,
+}: CheckboxProps & {
+  label: string
+}) {
+  return (
+    <View style={standardStyles.row}>
+      <Checkbox checked={checked} onPress={onPress} name={name} />
+
+      <Text style={[fonts.a, textStyles.medium, checkBoxStyles.labelArea]}>
+        <Label for={name} onPress={onPress} style={checkBoxStyles.label}>
+          {label}
+        </Label>
+      </Text>
+    </View>
+  )
+}
+
+function Checkbox({ checked, onPress, name }: CheckboxProps) {
+  return (
+    <View style={checkBoxStyles.border}>
+      <Text
+        style={[
+          checkBoxStyles.checkMark,
+          checked ? checkBoxStyles.checkMarkChecked : checkBoxStyles.hidden,
+        ]}
+      >
+        ✓
+      </Text>
+      {createElement('input', {
```

Comment:
Is this considered unstable? https://github.com/necolas/react-native-web/issues/1385

Reply:
oh no, 

we use this in other places too.

Reply:
given that there are other places we will have to fixes this when upgrading RNW and that Checkbox used this before and was just moved, id like to ask that we merge in for now. 

But good to know.

Reply:
Ok 👍